docs/developer-guide/working-with-plugins.md

```diff
@@ -140,6 +140,11 @@ ruleTester.run("custom-plugin-rule", rule, {
 });
 ```
 
+#### Customizing RuleTester
+
+RuleTester internally uses Mocha's `describe` and `it` methods when available to create tests for each valid and invalid case. If you use another test runner, you can override `RuleTester.describe` and `RuleTester.it` to make RuleTester compatible with it and have proper individual tests and feedback.
```

Comment:
For what it's worth, I would prefer "internally" to be explicit here. But I also like a lot of words to be explicit, possibly more than the average developer. Without "internally", I might assume a stronger dependency link (until I get to "when available" six words later).